src/core/constraint_template.jl

```diff
@@ -201,3 +219,81 @@ function constraint_tp_storage_exchange(pm::GenericPowerModel, i::Int; nw::Int=p
     PMs.constraint_storage_complementarity(pm, nw, i)
     constraint_tp_storage_loss(pm, nw, i, storage["storage_bus"], storage["r"], storage["x"], storage["standby_loss"])
 end
+
+
+"Links the voltage at both windings of a fixed tap transformer."
+function constraint_tp_trans_voltage(pm::GenericPowerModel, i::Int; nw::Int=pm.cnw)
```

Comment:
The constraints are split in two parts: constraints on the voltage, and constraints on the flow (on both power and current depending on the formulation). These could in principle all be merged in one constraint. But since the voltage constraints can be imposed without reference to the other variables, we prefer this separation.